test/spec_request.rb

```diff
@@ -227,7 +233,7 @@ class RackRequestTest < Minitest::Spec
 
     req = make_request \
       Rack::MockRequest.env_for("/", "HTTP_HOST" => "localhost:81", "HTTP_X_FORWARDED_HOST" => "2001:db8:cafe::17")
-    req.host_with_port.must_equal "2001:db8:cafe::17"
+    req.host_with_port.must_equal "[2001:db8:cafe::17]"
```

Comment:
As with `host`, `host_with_port` now using surrounding brackets for IPv6 should be specifically mentioned as breaking the CHANGELOG.

Reply:
Done.